src/modules/ekf2/EKF/ekf.cpp

```diff
@@ -281,23 +281,65 @@ void Ekf::predictState(const imuSample &imu_delayed)
 	_height_rate_lpf = _height_rate_lpf * (1.0f - alpha_height_rate_lpf) + _state.vel(2) * alpha_height_rate_lpf;
 }
 
-void Ekf::resetGlobalPosToExternalObservation(double lat_deg, double lon_deg, float accuracy, uint64_t timestamp_observation)
+void Ekf::resetGlobalPosToExternalObservation(double lat_deg, double lon_deg, float accuracy,
+		uint64_t timestamp_observation)
 {
+
 	if (!_pos_ref.isInitialized()) {
+		ECL_INFO("Global position reference not initialized, set home position first");
```

Comment:
```suggestion
		ECL_INFO("Global position reference not initialized, set home position first");
```

At a minimum this should be a warning, and then actually return a status so that the vehicle_command (top level ekf2) can send an acknowledgement back to the user/GCS. Otherwise this is effectively failing silently from a user perspective.